src/workerd/api/pyodide/pyodide.h

```diff
@@ -176,15 +203,20 @@ struct MemorySnapshotResult {
 // CPU architecture-specific artifacts. The logic for loading these is in getArtifacts.
 class ArtifactBundler: public jsg::Object {
 public:
+  kj::Maybe<const PyodidePackageManager&> packageManager;
+  // ^ lifetime should be static since there is normally one worker set for the whole process. see worker-set.h
```

Comment:
Or in more detail the ownership structure goes:
`WorkerSet` lifetime = `PackageManager` lifetime and `Worker` lifetime = `ArtifactBundler` lifetime and `WorkerSet` owns and will outlive `Worker`, so `PackageManager` outlives `ArtifactBundler`